sdk/ml/azure-ai-ml/tests/test_configs/internal/component_with_additional_includes/additional_includes_merge_folder.yml

```diff
@@ -0,0 +1,41 @@
+$schema: https://componentsdk.azureedge.net/jsonschema/CommandComponent.json
```

Comment:
we'd better make the component spec a valid one to submit? maybe you can use this spec instead: https://github.com/Azure/DesignerPrivatePreviewFeatures/blob/master/azure-ml-components/samples/components/additional-includes/component_entry/component_spec.yaml

Reply:
yes, found this is very annoying in yesterday's test, will update this and related test YAML

Reply:
updated additional includes' YAMLs to supported schema